repose-aggregator/components/filters/valkyrie-authorization/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala

```diff
@@ -136,10 +136,11 @@ class ValkyrieAuthorizationFilter @Inject()(configurationService: ConfigurationS
     def authorizeDevice(permissions: ValkyrieResult, deviceId: Option[String]): ValkyrieResult = {
       def authorize(deviceId: String, permissions: UserPermissions, method: String): ValkyrieResult = {
```

Comment:
I thought that was odd, but I left it alone since Adrian last touched those lines.  Is it worth changing?